Jenkinsfile

```diff
@@ -319,17 +504,13 @@ pipeline {
                         configFileProvider([configFile(fileId: 'maven-settings-nexus-zl', variable: 'MAVEN_SETTINGS')]) {
                             sh """\
                                #!/usr/bin/env bash
-                               bash .jenkins/scripts/release.sh ${env.BRANCH_NAME} ${env.PROJECT_VERSION} 
+                               bash .jenkins/scripts/release.sh ${env.BRANCH_NAME} ${qualifiedVersion} 
```

Comment:
release script is not modified and this script calls the docker images creation.